client/webserver/site/src/js/charts.js

```diff
@@ -277,11 +281,22 @@ export class DepthChart extends Chart {
     this.resize(parent.clientHeight)
   }
 
+  // setZoomBttns creates new regions for zoom in and zoom out buttons. It is
+  // used in initiation of the buttons and resizing.
+  setZoomBttns () {
+    this.zoomInBttn = new Region(this.ctx, new Extents(0, 0, 0, 0))
+    this.zoomOutBttn = new Region(this.ctx, new Extents(0, 0, 0, 0))
+  }
+
+  // initRegions sets up the zoom button regions.
+  initRegions () {
+    this.setZoomBttns()
+  }
```

Comment:
Seems like you can get rid one of these two methods now.

Reply:
Ok.